theano/sandbox/linalg/ops.py

```diff
@@ -1,66 +1,34 @@
 from __future__ import absolute_import, print_function, division
 import logging
-
-logger = logging.getLogger(__name__)
```

Comment:
@lamblin , I know the ```logger``` in here generates flake8 errors, but wouldn't moving it down miss logging the imports?